docs/CHANGELOG.rst

```diff
@@ -22,6 +22,8 @@ Added
 Changed
 -------
 - Refactor linters, check PEP 8 and PEP 257
+- Split expression engines into expression engines and execution engines
+- Use Jinja2 instead of Django Template syntax
```

Comment:
Because previously, we had things like `templatetags`, which were used in processes. This looked quite hacky to me. Now, filters for use in process definition expressions are defined using separate modules:
```python
FLOW_EXPRESSION_ENGINES = [
    {
        'ENGINE': 'resolwe.flow.expression_engines.jinja',
        'CUSTOM_FILTERS': [
            'resolwe_bio.expression_filters.sample',
        ]
    },
]
```